signac_dashboard/dashboard.py

```diff
@@ -590,7 +591,7 @@ def __call__(self, environ, start_response):
         """Call the dashboard as a WSGI application."""
         return self.app(environ, start_response)
 
-    def main(self):
+    def main(self, test_args=None):
```

Comment:
I would rather see this accept `*command_args`. `test_args` is a little odd because this is a public API. Docstring changes are also needed.
```suggestion
    def main(self, *command_args):
```
Then
```python
if len(command_args) == 0:
    command_args = None
```

Reply:
Not sure how to document this argument